ci/etcd.rb

```diff
@@ -0,0 +1,49 @@
+require './ci/common'
+
+namespace :ci do
+  namespace :etcd do |flavor|
+    task :before_install => ['ci:common:before_install']
+
+    task :install => ['ci:common:install'] do
+      sh %(wget --quiet https://github.com/coreos/etcd/releases/download/v2.0.3/etcd-v2.0.3-linux-amd64.tar.gz)
+      sh %(tar -xzvf etcd-* > /dev/null)
+      sh %(rm etcd-*.tar.gz)
+      sh %(mv `ls | grep etcd` etcd)
```

Comment:
Could you look at other ci files like apache, to use $VOLATILE_DIR as a dir to extract and download tarballs (otherwise it makes it a mess to run the tests locally).
Also set etcd to be installed in its own integration dir.
Can definitely help you do that if needed.